### ISSUE TITLE
wrong conversion of current date

### ISSUE THREAD

**Issue by mojtaba-yekta** [aliab/Persian-Date-Picker-Dialog#22]
Hi,
today is 30th Mehr 1397 (1397/7/30) but when I click "today" button it goes to 29 Mehr 1397 (1397/7/29)
it seems the conversion of Gregorian to Persian Calendar has logical error in calculation

**Comment by mojtaba-yekta** [comment]
I tested `PersianCalendar` class and test passed successfully:

```
@Test
    public void testPersinCalendar(){
        ir.hamsaa.persiandatepicker.util.PersianCalendar persianCal = new ir.hamsaa.persiandatepicker.util.PersianCalendar();
        persianCal.set(2018, Calendar.OCTOBER, 22);
        assertEquals(persianCal.getPersianShortDate(),"1397/07/30");
    }
```

**Comment by aliab** [comment]
Thank you for your report, this issue has been fixed in version 1.5.0
